README.md

```diff
@@ -1,6 +1,10 @@
 # Super Zombie Fortress [![Action Status](https://github.com/redsunservers/SuperZombieFortress/workflows/Package/badge.svg)](https://github.com/redsunservers/SuperZombieFortress/actions?query=branch%3Amaster)
 
-Custom Team Fortress 2 Gamemode, inspired from Left 4 Dead
+Custom Team Fortress 2 Gamemode, inspired by Left 4 Dead.
+
+There's 2 teams, Survivors (RED) and Zombies (BLU). The Survivors start with melee weapons and must survive for x amount of time while collecting weapons/ammo that are found around the map.
+
+The Zombie team starts off with a percent of the total amount of players, and must kill and infect the survivors before x amount of time runs out. To counteract the imbalance between melee-only and the ranged weapons that the survivors have, zombies can also "mutate" into special infected. On every kill, the zombie gains a special infected point, which is factored into spawning as a special infected after death.
```

Comment:
I'm no expert on english, but I'd add "all" on needing to infect all survivors.

Reply:
Oops, I missed that, will update that. Thanks